localfs/store.go

```diff
@@ -173,6 +173,10 @@ func (l *LocalStore) List(ctx context.Context, query cloudstorage.Query) (*cloud
 		if md, ok := metadatas[objname]; ok {
 			obj.metadata = md
 		}
+		if (query.StartOffset != "" && obj.name < query.StartOffset) ||
```

Comment:
I see you're making this exclusive for the StartingOffet, does that match the other stores?

Reply:
As I read/intend it, this treats StartOffset as an inclusive bound, and EndOffset as an exclusive bound. That matches my understanding of the [GoDoc on those fields](https://github.com/lytics/cloudstorage/blob/master/query.go#L13):

```
	StartOffset string   // (gcs only) "bar/", Only list objects lexicographically >= "bar/"
	EndOffset   string   // (gcs only) "foo/", Only list objects lexicographically < "foo/"
```

I suppose I should update those docs to say "gcs/localfs only" now, though.